caluma/mutation.py

```diff
@@ -127,12 +105,14 @@ def __init_subclass_with_meta__(
     @classmethod
     def get_serializer_kwargs(cls, root, info, **input):  # pragma: todo cover
         lookup_field = cls._meta.lookup_field
+        lookup_input_kwarg = cls._meta.lookup_input_kwarg
         model_class = cls._meta.model_class
 
         if model_class:
-            if "update" in cls._meta.model_operations and lookup_field in input:
+            if "update" in cls._meta.model_operations and lookup_input_kwarg in input:
                 instance = get_object_or_404(
-                    model_class, **{lookup_field: input[lookup_field]}
+                    model_class,
+                    **{lookup_field: extract_global_id(input[lookup_input_kwarg])}
```

Comment:
As discussed, add some documentation for `lookup_input_kwarg`

Reply:
Good point. I have added documentation to all possible meta options on `SerializerMutation`.